docs/index.rst

```diff
@@ -10,8 +10,8 @@ Solidity is an object-oriented, high-level language for implementing smart
 contracts. Smart contracts are programs which govern the behaviour of accounts
 within the Ethereum state.
 
-Solidity was influenced by C++, Python and JavaScript and is designed to target
-the Ethereum Virtual Machine (EVM).
+Solidity syntax-wise is a `curly-bracket language <https://en.wikipedia.org/wiki/List_of_programming_languages_by_type#Curly-bracket_languages>`_.
```

Comment:
Alt: "Solidity is part of the curly-bracket language family."

Reply:
do we need the word "syntax-wise"?

Reply:
Eu quero